pkg/config/types/requester.go

```diff
@@ -19,6 +19,7 @@ type RequesterConfig struct {
 
 	EvaluationBroker EvaluationBrokerConfig `yaml:"EvaluationBroker"`
 	Worker           WorkerConfig           `yaml:"Worker"`
+	S3SignedURL      S3SignedURLConfig      `yaml:"S3SignedURL"`
```

Comment:
We should _eventually_ move this into StorageConfig before we start adding more configs here.

Reply:
To understand better, are you suggesting something like:
```
type RequesterConfig struct {
	...
	
	StorageConfig    StorageConfig          `yaml:"StorageConfig"`
}


type StorageConfig struct {
	S3SignedURLConfig S3SignedURLConfig `yaml:"S3SignedURLConfig"`
}

type S3SignedURLConfig struct {
	Disabled   bool     `yaml:"S3SignedURLDisabled"`
	Expiration Duration `yaml:"S3SignedURLExpiration"`
}

Reply:
Yes, just so that requester doesn't end up with lots of top level configs for different providers.

Reply:
changed in https://github.com/bacalhau-project/bacalhau/pull/2995/commits/cbfb539d6dd5e6c954e411afc5c452030702d90a. The config structure now looks like:
```
    Requester:
        ...
        Worker:
            WorkerCount: 8
            WorkerEvalDequeueTimeout: 5s
            WorkerEvalDequeueBaseBackoff: 1s
            WorkerEvalDequeueMaxBackoff: 30s
        StorageProvider:
            S3:
                PreSignedURLDisabled: false
                PreSignedURLExpiration: 30m0s
```

Let me know if you have other thoughts, specially around `PreSignedURLDisabled`. I don't like the name, but I wanted to make it enabled by default

Reply:
No, that looks good, seems best to default to on.  And apologies I didn't realise it'd lead to renaming the other config structs (jobstore) too :D 

Looks good, we should merge....